components/arm/xarm/xarm.go

```diff
@@ -242,6 +242,10 @@ func (x *xArm) DoCommand(ctx context.Context, cmd map[string]interface{}) (map[s
 	if err = x.setGripperMode(ctx, false); err != nil {
 		return nil, err
 	}
+
```

Comment:
Can we wrap the above few lines in a command as well? Setting speed with a DoCommand should not fail if no gripper is attached.